bin/dax_tools/GenerateProcessorTemplate

```diff
@@ -141,8 +142,8 @@ class Processor_{name}(ScanProcessor):
 
 SESSION_LEVEL_TEMPLATE = DEFAULT_TEMPLATE+"""
 # Format for the spider command line
-SPIDER_FORMAT = '''python {{spider}} -v --spm {{spm}} \
--p {{proj}} -s {{subj}} -e {{sess}} -d {{dir}} --suffix "{{suffix_proc}}"'''
+SPIDER_FORMAT = '''python {{spider}} -p {{proj}} -s {{subj}} -e {{sess}}\
```

Comment:
I'm just curious and I would check this if I was in front of a computer,
but how is path settings handled? Is it part of the dax package or is it a
python settings file like the grid engine file?
On Oct 2, 2015 1:56 PM, "Benjamin Yvernault" <notifications@github.com>
wrote:

> In bin/dax_tools/GenerateProcessorTemplate
> <https://github.com/VUIIS/dax/pull/21#discussion_r41055452>:
>
> > @@ -141,8 +142,8 @@ class Processor_{name}(ScanProcessor):
> >
> >  SESSION_LEVEL_TEMPLATE = DEFAULT_TEMPLATE+"""
> >  # Format for the spider command line
> > -SPIDER_FORMAT = '''python {{spider}} -v --spm {{spm}} \
> > --p {{proj}} -s {{subj}} -e {{sess}} -d {{dir}} --suffix "{{suffix_proc}}"'''
> > +SPIDER_FORMAT = '''python {{spider}} -p {{proj}} -s {{subj}} -e {{sess}}\
>
> Yes for the space. You can remove the pathsettings. I thought I remove all
> of them. Ben
> … <#15029ea0a8c6db83_>
> On Oct 2, 2015, at 19:48, Brian D. Boyd <notifications@github.com> wrote:
> In bin/dax_tools/GenerateProcessorTemplate: > @@ -141,8 +142,8 @@ class
> Processor_{name}(ScanProcessor): > > SESSION_LEVEL_TEMPLATE =
> DEFAULT_TEMPLATE+""" > # Format for the spider command line >
> -SPIDER_FORMAT = '''python {{spider}} -v --spm {{spm}} \ > --p {{proj}} -s
> {{subj}} -e {{sess}} -d {{dir}} --suffix "{{suffix_proc}}"''' >
> +SPIDER_FORMAT = '''python {{spider}} -p {{proj}} -s {{subj}} -e {{sess}}\
> need a space before the \ — Reply to this email directly or view it on
> GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/VUIIS/dax/pull/21/files#r41055452>.
>